src/main/java/teammates/ui/template/InstructorFeedbackResultsQuestionTable.java

```diff
@@ -51,22 +49,15 @@ public InstructorFeedbackResultsQuestionTable(InstructorFeedbackResultsPageData
         this.questionStatisticsTable = questionStatisticsHtml;
         this.responses = responseRows;
         
-        this.isQuestionHasResponses = !responses.isEmpty(); //TODO: just check is response is empty in jsp? 
+        this.isQuestionHasResponses = isQuestionHasResponses;  
         
         this.question = question;
         
-        this.questionText = data.getBundle().getQuestionText(question.getId());
+        this.questionText = questionText;
         
-        this.panelClass = responses.isEmpty() ? 
-                          "panel-default" : 
-                          "panel-info";
+        this.panelClass = "panel-info";
```

Comment:
What happened to `"panel-default"`? Not being used anymore? Or was it just never used in the question view?